scripts/ci-configmap.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+
+CM_NAMES=("calico-addon" "calico-ipv6-addon" "calico-dual-stack-addon" "calico-windows-addon")
+CM_FILES=("calico.yaml" "calico-ipv6.yaml" "calico-dual-stack.yaml" "windows/calico")
+for i in "${!CM_NAMES[@]}"; do
+	if ! kubectl get configmap "${CM_NAMES[i]}" > /dev/null 2>&1 ; then
```

Comment:
Now that we're doing a kubectl apply we don't need to check for existence, in face we shouldn't otherwise the configmap won't be updated if changes are made to it

Reply:
Oh, I forgot to remove the if statement. Thank you for reminding. I have updated the code

Reply:
@CecileRobertMichon shall we merge this now?